### ISSUE TITLE
Create PDF after form submit

### ISSUE THREAD

**Issue by jakobroehrl** [nextcloud/forms#103]
It would be nice if forms could create a PDF after each submit. This PDF should contain the entered data. 
With that, it's easy to ask for declaration of consent or and store the answers automatically

**Comment by adamduncan** [comment]
Similarly, is there any way to alter the onSubmit action or event, in order to call custom functionality or redirect users to a different URL?

Doesn't seem any submit handlers are documented or exposed to consumers?

**Comment by jancborchardt** [comment]
@adamduncan that seems to be a different topic – please open a separate issue about that. :)

**Comment by NoxInmortus** [comment]
that would be a great option to have

**Comment by kevin147147** [comment]
Is somebody working on PDF export? Otherwise i'm thinking about implementing it. With tcpdf for example.

**Comment by NoxInmortus** [comment]
@kevin147147 Please do!

**Comment by jancborchardt** [comment]
@kevin147147 cool, feel free to go ahead! :)

**Comment by ei8fdb** [comment]
It would also be nice to be able to create a blank PDF of the form, with all answer options. 

This would allow for offline completion and post-data entry online.

**Comment by 089** [comment]
Good idea! A solution to convert templates (e.g. odt, docx) to PDF would also be great. I was thinking of something like https://carbone.io/ or https://github.com/PHPOffice/PHPWord

**Comment by pippcat** [comment]
Hey there. Are there any plans to implement this feature since the issue is stale since >2 years? It would be very useful, our small company f.e. is using Nextcloud forms for an event registration and we would love to have a way to automatically put the form data into some ODT or PDF template which users then can download. 
LibreOffice has this[ form letter](https://help.libreoffice.org/6.4/en-US/text/swriter/guide/form_letters_main.html) feature which allows to do it somehow but unfortunately CollaboraOnline isn't capable of doing it and even if it would it's still a manual process and annoying. :disappointed:

**Comment by nicovd737** [comment]
Hello,

My users are also complaining about this CSV export and they would like to have the offer to have a PDF export with the graphs inside like it's presented in the form result page.

Thanks !
Nico

**Comment by joesch2** [comment]
In my opinion, it would also be very helpful if the user could choose, after submitting the form, in which of his folders the PDF export with the content of the submitted form should be saved.
